DNN Platform/Website/Providers/DataProviders/SqlDataProvider/09.06.02.SqlDataProvider

```diff
@@ -0,0 +1,42 @@
+IF NOT EXISTS (SELECT * FROM sys.indexes WHERE object_id = OBJECT_ID(N'{databseOwner}.[{objectQualifier}ContentItems_MetaData]') AND name = N'IX_{objectQualifier}ContentItems_MetaData_MetaDataID')
+CREATE NONCLUSTERED INDEX [IX_{objectQualifier}ContentItems_MetaData_MetaDataID] ON {databaseOwner}.[{objectQualifier}ContentItems_MetaData]
+(
```

Comment:
Doesn't `{databaseOwner}` include the `.`?
```suggestion
IF NOT EXISTS (SELECT * FROM sys.indexes WHERE object_id = OBJECT_ID(N'{databaseOwner}[{objectQualifier}ContentItems_MetaData]') AND name = N'IX_{objectQualifier}ContentItems_MetaData_MetaDataID')
CREATE NONCLUSTERED INDEX [IX_{objectQualifier}ContentItems_MetaData_MetaDataID] ON {databaseOwner}[{objectQualifier}ContentItems_MetaData]
(
```

Reply:
Oops, darn auto updating scripts